Data/Migrations/20200904113526_DecriptionAddedForHighlightTable.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Are these generated by efcore migration tool? Not familiar enough with this to review it.

Reply:
Yes, if any changes are made in the database (models) you need to create a migration. This tool is used to create it.

You can find some documentation about it here:
https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/?tabs=dotnet-core-cli